doc/source/comparison_with_sql.rst

```diff
@@ -372,10 +372,123 @@ In pandas, you can use :meth:`~pandas.concat` in conjunction with
 
     pd.concat([df1, df2]).drop_duplicates()
 
+SOME ANALYTIC AND AGGREGATE FUNCTIONS
+-------------------------------------
+Top N rows with offset
+
+.. code-block:: sql
+
+    -- MySQL
+    SELECT * FROM tips
+    ORDER BY tip DESC
+    LIMIT 10 OFFSET 5;
+
+    -- Oracle 12c+
+    SELECT * FROM tips
+    ORDER BY tip DESC
+    OFFSET 5 ROWS FETCH NEXT 10 ROWS ONLY;
+
+In pandas:
+
+.. ipython:: python
+
+    tips.sort_values(['tip'], ascending=False).head(10+5).tail(10)
```

Comment:
use idiomatic pandas if you can (and don't sort if not needed)

``tips.nsmallest(10, columns='tip').tail(5)``